### DISCUSSION TITLE
[Feature] Remove images from detected face or move some images to another face

### DISCUSSION THREAD

**Discussion by jakob42** [immich-app/immich#4097] [Feature Request]
### The feature

Hi,

generally the facial recognition works quite well for me. I just stumbled about a photo from my son as a baby (literally a few hours old :)) and it identified pictures from my son from the next couple of years. Amazing. But it also mixed in a few wrong ones, not that I can blame it, for example his younger sister appears there as well. Before I merge them, I would like to remove the wrong ones. Is that possible? Couldn't find it and sounds like a useful feature. While the detection is good, it's not perfect.

Even better would be the option to move the wrongly detected images directly into another recognized face.

Thanks for considering.
(Copied this from discord)

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #2472

**Comment by jakob42** [comment]
Thanks, saw this but I missed nr 10 :+1:
